api/docs/release.dox

```diff
@@ -165,6 +161,10 @@ compatibility changes:
    as part of AVX-512, e.g. xmm16 - xmm31.
  - Dropped support for clients used with statically linked DynamoRIO to reach
    the code cache with 32-bit displacements.
+ - An additional parameter in the accessors to the cache stats that passes
```

Comment:
I think this needs some context: there are many caches in DR in general: its code cache, trace cache, persistent caches, etc.  Best to at least mention that this is inside drcachesim, and I would list the classes involved by precise name.

Reply:
Done.